pages/pipelines.md

```diff
@@ -94,4 +94,4 @@ Job exit status may include the exit signal reason, which indicates whether the
 <%= image "exit-status.png", width: 2048/2, height: 880/2, alt: "Exit status of a job" %>
 
 >🚧
-> Exit status information available in the <a href="/docs/apis/graphql-api">GraphQL API</a> but not the <a href="/docs/apis/rest-api">REST API</a>.
+> Exit status information is only available using the <a href="/docs/apis/graphql-api">GraphQL API</a>.
```

Comment:
Possible alternatives

```suggestion
> Exit status information is only available in the <a href="/docs/apis/graphql-api">GraphQL API</a>, not the <a href="/docs/apis/rest-api">REST API</a>.
```

```suggestion
> Exit status information is only available in the <a href="/docs/apis/graphql-api">GraphQL API</a>.
```

```suggestion
> Exit status information is not available in the <a href="/docs/apis/rest-api">REST API</a>.
```

Reply:
I don't think this warrants the callout treatment. Callouts have been used fairly liberally as quick solutions but using too many undermines the ones that are actually important. How about we just make this regular text, like:

```
If you want to access the exit status through an API, it's only available in the [GraphQL API](/docs/apis/graphql-api).
```

What do you think? (I couldn't add a suggestion across the lines 😔)

Reply:
Agreed!